test/exe/Main.hs

```diff
@@ -430,13 +430,13 @@ codeLensesTests = testGroup "code lenses"
 watchedFilesTests :: TestTree
 watchedFilesTests = testGroup "watched files"
   [ testSession "workspace file" $ do
-      _ <- openDoc' "A.hs" "haskell" "module A where"
+      _ <- openDoc' "C/A.hs" "haskell" "module A where"
```

Comment:
Why did this change?

Reply:
This is an incomplete attempt at making a test more demanding. Will try to push an improved version tonight.

Reply:
pushed